Services/Init/classes/class.ilInitialisation.php

```diff
@@ -572,14 +572,9 @@ protected static function initDatabase()
      */
     public static function setSessionHandler()
     {
-        if (ini_get('session.save_handler') != 'user' && version_compare(PHP_VERSION, '7.2.0', '<')) {
-            ini_set("session.save_handler", "user");
-        }
-
-        require_once "Services/Authentication/classes/class.ilSessionDBHandler.php";
         $db_session_handler = new ilSessionDBHandler();
         if (!$db_session_handler->setSaveHandler()) {
-            self::abortAndDie("Please turn off Safe mode OR set session.save_handler to \"user\" in your php.ini");
+            self::abortAndDie("Please set session.save_handler to \"user\" in your php.ini");
```

Comment:
Maybe a more generic error message: something like "Cannot start session handling."

Reply:
Changed with the latest commit